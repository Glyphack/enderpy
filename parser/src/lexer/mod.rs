mod diagnostics;
pub mod fstring;
pub mod lexer;
