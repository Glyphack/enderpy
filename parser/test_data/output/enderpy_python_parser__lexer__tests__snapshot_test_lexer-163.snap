---
source: parser/src/lexer/mod.rs
description: "a = rb\"hello\""
---
0,1: Identifier                  (Str("a"))
2,3: =                                        (None)
4,13: RawBytes           (Str("rb\"hello\""))
